doc/book/in-depth-guide/first-module.md

```diff
@@ -197,6 +207,39 @@ We now need to tell our module where to find this controller named
 `module/Blog/config/module.config.php`.
 
 ```php
+=======
+No Exception available
+~~~~
+
+We now need to tell our module where to find this controller named `Blog\Controller\ListController`. To achieve this we have to add this key to the `controllers` configuration key inside your `/module/Blog/config/module.config.php`.
+
+~~~~ {.sourceCode .php}
```

Comment:
Conflict markers are still present in your version, @samsonasik , indicating a failed merge.

If you look at the current version, I've already done all the changes you suggested in your original PR when I pushed yesterday, with a ton more (I essentially rewrote this entire tutorial).

Pull latest master and start over if you want to look for things to correct (there shouldn't be much at this point).